code/__DEFINES/misc.dm

```diff
@@ -141,7 +141,7 @@
 //2 = reduced hud (just hands and intent switcher)
 //3 = no hud (for screenshots)
 
-#define MINERAL_MATERIAL_AMOUNT 2000
+#define MINERAL_MATERIAL_AMOUNT 1000
```

Comment:
no everything used to be 3750

then we changed it to 2000, but everything was hardcoded for 3750, which caused problems

so we fixed it to not hardcode and work off this define

meaning shit won't break core, stop being a dipshit